tests/integration/test_notebooks.py

```diff
@@ -24,18 +24,26 @@
 from nbconvert.preprocessors import ExecutePreprocessor
 from nbconvert.preprocessors.execute import CellExecutionError
 
+
+NOTEBOOK_DIR = "../../doc/source/notebooks"
+
+
+def _nbpath():
+    this_dir = os.path.dirname(__file__)
+    return os.path.join(this_dir, NOTEBOOK_DIR)
```

Comment:
I think `NOTEBOOK_DIR` variable is unnecessary here. Could you return a string back and remove `NOTEBOOK_DIR`, please?

Reply:
I think it's better to have config-style things like that as global variables at the top of the file rather than hidden inside some helper function...